demo-app/src/CreateShape.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from "react";
+import ClipPathPreviewForm from "./ClipPathPreviewForm.js";
+import ClipPathPreview from "./ClipPathPreview.js";
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+
+const CreateShape = (props) => {
+  
+  // Change these keys to the ones in schema.js
+  const [formInputs, setFormInputs] = useState({
+    //Schema Variables
+    "name": "Square", 
+    "type": "square",
+    "formula": "polygon(10% 10%, 90% 10%, 90% 90%, 10% 90%)",
+    "vertices": "4",
+    "edges": "4", 
+    "notes": "A circle is a round shaped figure that has no corners or edges. In geometry, a circle can be defined as a closed, two-dimensional curved shape.", 
+    //Other Variables
+    "clipPathType": "polygon",
+    "showShadow": false, 
+    "backgroundColor": "#12a8d6",
```

Comment:
`name`: Tilted Square 
`notes`  as empty(`''`). 
Make the formula as, `polygon(10% 10%, 90% 10%, 90% 90%, 10% 80%)`.

Reply:
Changed!

Reply:
Thanks, resolving.